openapi/consumed/api-piattaforma-notifiche.yaml

```diff
@@ -555,6 +555,9 @@ components:
         url:
           minLength: 1
           type: string
+        category:
+          type: string
```

Comment:
why not enforcing some constraints? min max length (3 - 30), a regex to expect just UPPERCASE_SNAKE_CASED_VALUES, with a right decoder to check this when it moves inside the backend?